src/notation/internal/notationparts.cpp

```diff
@@ -1055,15 +1054,12 @@ void NotationParts::sortParts(const PartInstrumentList& parts, const QList<Ms::S
 
             trackMapping.append(originalStaves.indexOf(staff));
             staffMapping.append(actualStaffIndex);
-            sortingNeeded |= actualStaffIndex != runningStaffIndex;
             ++runningStaffIndex;
         }
         ++partIndex;
     }
 
-    //if (sortingNeeded) {
```

Comment:
I think this is a wonderful opportunity to ask @asattely why he disabled this code, and @njvdberg why this code is needed :)

Reply:
@RomanPudashkin Needed is maybe too much but this was just an optimization to prevent an unnecessary instance on the undo stack. Apparently the implementation of system objects might require always sorting but that's what @asattely can comment on.